app/templates/partials/introduction/preview.html

```diff
@@ -1,8 +1,8 @@
-<h3 class="u-fs-m">{{ intro.title }}</h3>
+<h3 class="u-fs-l">{{ intro.title }}</h3>
```

Comment:
Actually, the original change here was correct I think it just wasn't captured in the ticket. The font-size should be `u-fs-m`.

Reply:
Ok - will revert back for preview, do you want the similar change in basic.html also reverted?

Reply:
No that's fine